ci-operator/config/openshift/cluster-operator/master.json

```diff
@@ -0,0 +1,89 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.11",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_images": {
+    "root": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "release",
+      "tag": "golang-1.10"
+    },
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "base"
+    },
+    "ansible": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "ansible"
+    },
+    "test_base_image": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "release",
+      "tag": "golang-1.10"
+    }
+  },
+  "canonical_go_repository": "github.com/openshift/cluster-operator",
+  "binary_build_commands": "NO_DOCKER=1 make build",
+  "images": [
+    {
+      "from": "base",
+      "to": "cluster-operator",
+      "dockerfile_path": "build/cluster-operator/Dockerfile",
+      "inputs": {
+        "bin": { "paths": [{ "source_path": "/go/src/github.com/openshift/cluster-operator/bin/cluster-operator", "destination_dir": "." }] }
+      }
+    },
+    {
+      "from": "ansible",
+      "to": "cluster-operator-ansible",
+      "context_dir": "build/cluster-operator-ansible"
```

Comment:
I didn’t realize we were doing this :(

Reply:
Eventually we won't run ansible to do stuff, since we're going to be invoking the installer directly. Because this image will eventually go away, it doesn't make much sense to try to include its changes into the openshift-ansible repo.